src/main/java/org/kainos/ea/cli/ClientName.java

```diff
@@ -0,0 +1,17 @@
+package org.kainos.ea.cli;
+
+public class ClientName {
+    private String name;
+
+    public ClientName(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+}
```

Comment:
could we just return a string from the endpoint rather than create this new object?